slick-testkit/src/codegen/resources/dbs/uuid-h2.sql

```diff
@@ -1,3 +1,5 @@
 create table "person" ("id" INTEGER NOT NULL PRIMARY KEY,
```

Comment:
@cvogt the `uuid.sql` was renamed to `uuid-h2.sql`, do we want to keep `uuid.sql`?